src/_assets/js/main.js

```diff
@@ -22,7 +22,7 @@ $(function () {
   setupClipboardJS();
 
   // New (dash) tabs
-  setupTabs($('.nav-tabs'), 'io.flutter.tool-id');
```

Comment:
#editor-setup was here previously right? Is that using the new tag too? If not, are we breaking it?

Reply:
Where is the #editor-setup code example?

Reply:
https://github.com/flutter/website/blob/master/src/_assets/js/main.js#L25